tests/assets/recipes/parametric-daylight-factor/flow/main.yaml

```diff
@@ -0,0 +1,47 @@
+name: main
+inputs:
+  parameters:
+  - name: model-names
+    descriptions: a list of model folder names
+  artifacts:
+  - name: models-folder
+    description: A folder full of honeybee models
+
+tasks:
+
+- name: daylight-factor
+  template: daylight-factor
+  loop:
+    from:
+      type: inputs
+      variable: model-names
+  arguments:
+    artifacts:
+    - name: model
+      from:
+        type: inputs
+        variable: models-folder
+      subpath: '{{item}}/model'
```

Comment:
nice trick here! Is it supported for all artifacts? Does it mean that this artifact will always be copied under this subfolder?

Reply:
This is the inputs artifacts. This was used as syntactic sugar to make it possible to set paths dynamically through a parameter. It is mostly useful for loops as demoed above.